e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/k8s-percona-openebs-pod-cleanup.yml

```diff
@@ -1,41 +1,69 @@
-- hosts: localhost
 
-  vars_files:
-    - openebs-ns-vars.yml
 
-  tasks:
-       
+   
        - name: Delete percona mysql pod
-         shell: source ~/.profile; kubectl delete -f "{{ openebs_percona }}"
+         shell: source ~/.profile; kubectl delete -f "{{ percona_file }}" -n percona
          args:
            executable: /bin/bash
          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
        - name: Confirm percona pod has been deleted
-         shell: source ~/.profile; kubectl get pods
+         shell: source ~/.profile; kubectl get pods --all-namespaces -l name=percona 
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         register: result
+         until: "'percona' not in result.stdout"
+         delay: 30
+         retries: 10
+
+       - name: Get pvc name to verify successful pvc deletion
+         shell: source ~/.profile; kubectl get pvc --all-namespaces | grep {{ replace_with.0 }} | awk {'print $3'}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         register: pvc
```

Comment:
Can we also check that the pods created by the PV also are deleted - that is precisely the intent of this exercise of enhancing cleanups

Reply:
we are greping for pvc volume name not PVC name.
ubuntu@kubemaster-01:~/openebs/k8s/demo/percona$ kubectl get pvc | grep demo-vol1-claim | awk {'print $3'}
pvc-9781773a-46c8-11e8-af46-02b983f0a4db